classes/03class/exercises/c03-make01/gpavelar/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM httpd:2.4.41-alpine
+ARG REGISTRY
+ENV REGISTRY ${REGISTRY}
+ARG TAG
+ENV TAG ${TAG}
```

Comment:
You don't need to specify ENV variables in this case. Those variables will not be changed after image is built.

- **ARG:** variables that can be modified at building time only. via `docker build --build-args`
- **ENV:** variables that can be preset at building time and can be modified at runtime. via `docker run --env`

This image might help to understand:

![Docker ARG vs ENV](https://d33wubrfki0l68.cloudfront.net/8d799fc311d166c3a7d9f1a7970e1dd1d43bf141/0a855/images/docker-env-vars/docker_environment_build_args.png)

Reply:
@alanlima Thanks alot, I really understood it in a different way. I am getting used to it. So I have save this on my notes, but i will need some time to make those changes.